core/src/components/button/button.ionic.scss

```diff
@@ -43,8 +43,8 @@
 // --------------------------------------------------
 
 :host(.button-solid) {
-  --background-activated: #{globals.ion-color(primary, shade)};
-  --background-hover: #{globals.ion-color(primary, shade)};
+  --background-activated: #{globals.$ion-bg-primary-base-press};
+  --background-hover: #{globals.$ion-bg-primary-base-press};
```

Comment:
Why was this changed? `#{globals.ion-color(primary, shade)};` works. You can verify this by looking at the [ionic colors map](https://github.com/ionic-team/ionic-framework/blob/9399d46e2d813a2e247138e2c8e04a0416bd6a7d/core/src/themes/ionic/ionic.theme.default.scss#L34-L35).